backend/app/api/v1/endpoints/ocp/ocpJobs.py

```diff
@@ -33,6 +33,9 @@ async def jobs(
         examples=["2020-11-15"],
     ),
     pretty: bool = Query(False, description="Output contet in pretty format."),
+    size: int = Query(None, description="Number of jobs to fetch"),
+    offset: int = Query(None, description="Offset Number to fetch jobs from"),
+    sort: str = Query(None, descption="To sort fields on specified direction"),
```

Comment:
The `description` keyword here is spelled "descption". Does `Query` accept kwargs, or is the IDE showing an error here?? (The big problem with Python's free-form "kwargs" convention is that it doesn't allow an IDE to catch this sort of bug.)